apps/token-manager/contracts/TokenManager.sol

```diff
@@ -351,7 +351,7 @@ contract TokenManager is ITokenController, AragonApp { // ,IForwarder makes cove
     * @param _owner The address that sent the ether to create tokens
     * @return True if the ether is accepted, false for it to throw
     */
-    function proxyPayment(address _owner) payable public returns (bool) {
+    function proxyPayment(address _owner) payable isInitialized public returns (bool) {
```

Comment:
I think we don't need to have `isInitialized` on this, since it's not a `view` simply out of conformance to the minime interface.

Reply:
Ya, I was in doubt about these ones, but I thought it was more aligned with the purpose of the issue: some day we could change the functions to modify state. Actually, most of the changes in this PR are not needed. But I agree with your comment, so removing it.